docs/pages/machine-id/reference/cli.mdx

```diff
@@ -0,0 +1,72 @@
+   --data-dir=/var/lib/teleport/bot \
+   --destination-dir=/opt/machine-id \
+   --token=00000000000000000000000000000000 \
+   --join-method=token \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=auth.example.com:3025
+```
+</Details>
+
+| Flag                 | Description                                                                                    |
+|----------------------|------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                              |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                     |
+| `-a/--auth-server`   | Address of the Teleport Auth Server (On-Prem installs) or Proxy Server (Cloud installs).       |
+| `--token`            | A bot join token, if attempting to onboard a new bot; used on first connect.                   |
+| `--ca-pin`           | CA pin to validate the auth server.                                                            |
+| `--data-dir`         | Directory to store internal bot data. Access should be limited. See `tbot init` for details.   |
+| `--destination-dir`  | Directory to write short-lived machine certificates.                                           |
+| `--certificate-ttl`  | TTL of short-lived machine certificates.                                                       |
+| `--renewal-interval` | Interval at which short-lived certificates are renewed; must be less than the certificate TTL. |
+| `--join-method`      | Method to use to join the cluster, can be `token` or `iam`.                                    |
+| `--one-shot`         | If set, quit after the first renewal.                                                          |
+
+## `tbot init`
+
+Initialize a certificate destination directory for writes from a separate bot
+user. Useful when running the bot under an isolated user account.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=ubuntu \
+    --owner=ubuntu:ubuntu \
+    --reader-user=jenkins
+```
+
+| Flag                 | Description                                                                                           |
+|----------------------|-------------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                                     |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                            |
+| `--destination-dir`  | Directory to write short-lived machine certificates to.                                               |
+| `--owner`            | Defines Linux user:group owner of `--destination-dir`. Defaults to `nobody:nobody` if unspecified.    |
```

Comment:
Mentioned again below, but this string is no longer accurate. PR to fix that: https://github.com/gravitational/teleport/pull/11158

```suggestion
| `--owner`            | Defines Linux user:group owner of `--destination-dir`. Defaults to the bot user if unspecified.       |
```